src/routers/task.js

```diff
@@ -92,4 +94,47 @@ router.delete('/tasks/:id', auth, async(req, res) => {
     }
 })
 
+const upload = multer({
+    limits: {
+        fileSize: 1000000
+    },
+    fileFilter(req, file, callback) {
+        if (!file.originalname.match(/\.(jpg|jpeg|png)$/)) {
```

Comment:
use `path.extname` and `Array#includes`

Reply:
Excuse me, but where? Instead of what? :/

Reply:
instead of this line. `path.extname` and `Array#includes` is faster and it protects agains evil regex attack in future if regex is modified.

Reply:
Fixed.

Reply:
It is fixed, but because of code duplication not in the whole project  :sweat_smile:

Reply:
Fixed.
Moved the method into a separate one.